test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Agent/AgentTestUtils.cs

```diff
@@ -30,7 +30,7 @@ public static class AgentTestUtils
                 Id = 1,
                 JobName = job.Name,
                 StepName = stepName,
-                SubSystem = "T-SQL",
+                SubSystem = SqlServer.Management.Smo.Agent.AgentSubSystem.TransactSql,
```

Comment:
Would suggest adding a unit test that has a hardcoded list of all the currently known SMO enum types and verifying that the enum only contains those types so that we can catch newly added types ASAP (for both enums used here)